tests/unit_tests/core/crypto/aes.cc

```diff
@@ -35,6 +35,7 @@
 #include <boost/test/unit_test.hpp>
 
 #include "core/crypto/aes.h"
+#include "core/util/impl/log.cc"
```

Comment:
`#include`'ing implementation files is not necessary (and is a long deprecated practice). *If* we need to include an implementation file, it can be included in CMakeLists.txt - though we don't need to because we link with `libcore` which has the logger implementation already built.

Since I cannot reproduce a segfault, I'll need more information in #457.